### PR TITLE
Expanded the direct install and run instructions.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -190,10 +190,13 @@ For example, using Gunicorn:
...
     $ gunicorn httpbin:app
 
-Or run it directly:
+Or install and run it directly:
 
 .. code:: bash
 
+    $ git clone git@github.com:Runscope/httpbin.git
+    $ cd httpbin
+    $ [sudo] python setup.py install
```
Comment (kennethreitz) [inline README.rst]: no `[sudo]`

Comment (wdschei) [inline README.rst]: Why not? Without `sudo`'ing I got:
```
running install
error: can't create or remove files in install directory

The following error occurred while trying to add or remove files in the
installation directory:

    [Errno 13] Permission denied: '/usr/local/lib/python2.7/dist-packages/test-easy-install-6659.write-test'

The installation directory you specified (via --install-dir, --prefix, or
the distutils default setting) was:

    /usr/local/lib/python2.7/dist-packages/

Perhaps your account does not have write access to this directory?  If the
installation directory is a system-owned directory, you may need to sign in
as the administrator or "root" account.  If you do not have administrative
access to this machine, you may wish to choose a different installation
directory, preferably one that is listed in your PYTHONPATH environment
variable.

For information on other options, you may wish to consult the
documentation at:

  https://pythonhosted.org/setuptools/easy_install.html

Please make the appropriate changes for your system and try again.
```

Comment (sigmavirus24) [inline README.rst]: @wdschei for several reasons:

1. You should be using a virtualenv for local development/usage
2. You should be using `pip` to install it, e.g., `pip install -e .`
3. People who tend to read this documentation, are not deploying HTTPbin but running it locally and installing the package globally is a poor choice for them that will only cause numerous headaches in the future.

**Comment by wdschei** [comment]
@sigmavirus24 Thanks for the explanation.

File: README.rst (hunk 1 above)
Comment (wdschei) [inline README.rst]: @sigmavirus24 Thanks for the explanation.

Comment (sigmavirus24) [inline README.rst]: @wdschei would you update this PR to reflect that or do you just want to abandon this?

Comment (wdschei) [inline README.rst]: @sigmavirus24 Updated.

File: README.rst [hunk 2]
```diff
@@ -196,7 +196,7 @@ Or install and run it directly:
 
     $ git clone git@github.com:Runscope/httpbin.git
     $ cd httpbin
-    $ [sudo] python setup.py install
+    $ python setup.py install
```
Comment (sigmavirus24) [inline README.rst]: This should be `pip install -e .` as I explained in the previous discussion.

**Pull request merged**